src/lib/modules/Loan/backoffice/OverdueLoanSendMailModal/OverdueLoanSendMailModal.js

```diff
@@ -48,20 +48,20 @@ export default class OverdueLoanSendMailModal extends Component {
         <Modal.Content>
           <Modal.Description>
             <Header>
+              {'Loan on '}
               <Link
-                to={BackOfficeRoutes.loanDetailsFor(loan.metadata.pid)}
+                to={BackOfficeRoutes.documentDetailsFor(
+                  loan.metadata.document.pid
+                )}
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                Loan
-              </Link>
-              {' on '}
-              <Link
-                to={BackOfficeRoutes.itemDetailsFor(loan.metadata.item_pid)}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Item
+                {loan.metadata.document.title +
```

Comment:
There is a actually a component for this:
```suggestion
                <LiteratureTitle 
                    title={loan.metadata.document.title}
                    edition={loan.metadata.document.edition}
                    publicationYear={loan.metadata.document.publication_year} />
```